pkg/lib/authn/authenticator/password/checker.go

```diff
@@ -297,6 +312,9 @@ func (pc *Checker) PasswordRules() string {
 	if pc.PwLowercaseRequired {
 		builder.WriteString("required: lower; ")
 	}
+	if pc.PwAlphabetRequired {
+		builder.WriteString("required: alphabet; ")
```

Comment:
alphabet is not a builtin character class. https://developer.apple.com/documentation/security/password_autofill/customizing_password_autofill_rules

I think we need to use **Custom Character Class** here.

Reply:
@louischan-oursky What about using `special`?

Reply:
`special` is mapped to our `symbol` policy.